StratisCore.UI/src/app/wallet/send/send.component.ts

```diff
@@ -394,4 +394,30 @@ export class SendComponent implements OnInit, OnDestroy {
   private startSubscriptions() {
     this.getWalletBalance();
   }
+
+  private numberToString(num) {
+    let numStr = String(num);
+    if (Math.abs(num) < 1.0) {
+      let e = parseInt(num.toString().split('e-')[1]);
+      if (e) {
+        const negative = num < 0;
+        if (negative) {
+          num *= -1;
+        }
+        num *= Math.pow(10, e - 1);
+        numStr = '0.' + (new Array(e)).join('0') + num.toString().substring(2);
+        if (negative) {
+          numStr = '-' + numStr;
+        }
+      }
```

Comment:
we can return `numStr` here, that way there is no need for the subsequent `else` which reduces nesting